playwright.config.ts

```diff
@@ -77,8 +61,4 @@ const config = defineConfig<TestOptions>({
   reporter: process.env.CI ? 'github' : 'list',
 });
 
-if (process.env.CI) {
-  config.retries = 3;
-}
```

Comment:
@himself65 We should fix flaky tests (maybe with hacky waits), rather than ignoring them.